scripts/dataCreation.js

```diff
@@ -1,3 +1,11 @@
+const { User } = require('../app/models');
+
 exports.execute = () => {
   // This function should create data for testing and return a promise
+  return User.create({
+    name: 'Federico',
+    surname: 'Casares',
+    email: 'federico.casares@wolox.com.ar',
+    password: '12345678'
```

Comment:
This non-hashed password could bring you problems in future tests

Reply:
why?